inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/InlongStreamEntity.java

```diff
@@ -30,25 +30,24 @@ public class InlongStreamEntity implements Serializable {
 
     private static final long serialVersionUID = 1L;
     private Integer id;
-    private String inlongStreamId;
     private String inlongGroupId;
+    private String inlongStreamId;
     private String name;
     private String description;
-    private String mqResourceObj;
-    private String dataSourceType;
-    private Integer storagePeriod;
+    private String mqResource;
```

Comment:
Topic is configurated by inlong_group_id, not inlong_stream_id.

Reply:
For Pulsar, Topic is composed of group and stream.